controllers/auth.js

```diff
@@ -1,6 +1,22 @@
 const passport = require("passport");
 const users = require("../models/users");
 const authService = require("../services/authService");
+const REALDEVSQUAD_HOSTNAME = new URL(config.get("services.rdsUi.baseUrl")).hostname;
```

Comment:
Adding this constant here seems redundant as we already store a similar value variable `rdsUiUrl` on line 30.
We can get the hostname from it.